projects/sbb-esta/angular-public/breadcrumb/src/styles/_breadcrumb-base.scss

```diff
@@ -75,21 +85,35 @@
     display: flex;
   }
 
+  &:first-of-type a {
+    height: toPx($breadcrumbLabelHeight);
+  }
+
+  &:last-of-type a {
+    color: $sbbColorBlack;
+  }
+
   @include mq($from: tabletPortrait) {
     display: flex;
   }
 
   padding-left: 0;
-  height: toPx($breadcrumbHeight);
-  margin-right: toPx($breadcrumbArrowMarginsDT);
 
-  @include mq($from: desktop4k) {
-    margin-right: toPx($breadcrumbArrowMarginsDT * $scalingFactor4k);
-    height: toPx($breadcrumbHeight * $scalingFactor4k);
+  @if $sbbBusiness {
```

Comment:
use @if $sbbBusiness {... in _variables.scss and overwrite $breadcrumbArrowMarginsDT instead of creating a second var